File: submissions/sm_029_sakshi/week_3/day_3/session_2/average_divisible_while.js
function avgdiv(limit,number)
{
    var sum=0
    var count=0
    var i=0
    while(i<=limit)
    {
        if(i%number==0)
        {
            sum=sum+i
            count=count+1
          //console.log(sum)
           //console.log(count)
        }
      i++

    }
    return sum/count
}
avgdiv(20,4)
console.log(avgdiv(20,4))

File: submissions/sm_105_ashish/week_13/day_4/session_1/average_divisible_while.py
limit = int(input('Please enter the limit :'))
number = int(input('Please enter the number :'))
i = 0
sum = 0
count = 0
while i <= limit:
    if i % number == 0:
        sum = sum + i
        count = count + 1
    i = i + number   
print(sum/(count-1))
